src/style.css

```diff
@@ -0,0 +1,93 @@
+@import url(./styles/star.css);
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  line-height: 18px;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card__code {
+  color: #616070;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.wrapper {
+  margin-bottom: 24px;
+}
+
+.card__wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.wrapper__reviews {
+  font-size: 10px;
+  color: #060b35;
+  line-height: 14px;
+}
+
+.price {
+  margin-bottom: 16px;
+}
+
+.price__text {
+  font-size: 12px;
+}
+
+.price__number {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__buy {
+  background: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  width: 100%;
+  height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding instead